code/service/src/main/resources/application.conf

```diff
@@ -186,13 +183,28 @@ write-side-slick {
 
 jdbc-journal {
 	tables {
-		journal {
+		# Only used in pre 5.0.0 for backward-compatibility
+		# ref: https://github.com/akka/akka-persistence-jdbc/blob/v5.0.0/core/src/main/resources/reference.conf
+		legacy_journal {
 			tableName = "journal"
 			schemaName = ${jdbc-default.schema}
 		}
+
+		# this is the new going forward
+		# ref: https://github.com/akka/akka-persistence-jdbc/blob/v5.0.0/core/src/main/resources/reference.conf
+		event_journal {
+			tableName = "event_journal"
```

Comment:
will this create unused tables for now?

Reply:
Please check comment below

Reply:
At it is now if you pull the repo and build it locally you will only have the new schemas since I have removed the backward compatibility mechanism. Since we will be working the migration tool I think it is worth leaving it as it is.